### ISSUE TITLE
Add ability to hide My Publications

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#2811]
https://forums.zotero.org/discussion/comment/416757/#Comment_416757

Similar to Duplicate Items and Unfiled Items

**Comment by Xylol** [comment]
Not referring to the forum posts. 
But indeed I would like to see an option to hide "My Publications".

**Comment by Benature** [comment]
same issue

**Comment by jpwiega** [comment]
Agreed, should be an easy change to allow those of us not using this (or recreating a private version of the folder), to not waste folder real estate and/or have a redundant folder.  Thanks for considering.

**Comment by abatty** [comment]
I support this as well. I have always had a saved search called "My Publications," and I am not using the Zotero shared one. I'd like to hide it.

**Comment by GxDeWitt** [comment]
Refresh this issue.
